### PR TITLE
New Script: Flip Screen on External Monitor to dictated degree°

### PR DESCRIPTION
**Pull request by ybbond**
## Description

It is a new script to toggle display rotation on external monitor defined on `<display>`, to some defined `<degree>`.
- The `<display` name can be seen by open `System Preferences` → `Displays`, see the window name.
    - example: `DELL P2212H`
- The `<degree>` is a choice between `90`, `180` and `270`.

This script handles whether you **mirror** or **extends** your second display on external monitor.

## Type of change

- [x] New script command

## Screenshot

This script 'plays' with `System Preferences` of `Displays` section.

Screenshot of System Preferences on Display scope that this script modifies.

Demo (2.8 MegaBytes, 16 seconds GIF):

Demo of the Flip Screen Demo

## Dependencies / Requirements

No dependency 😄 

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ybbond** [comment]
hmmm... the diff file detected as binary file 😕

**Comment by RSO** [comment]
Hey @ybbond! Thank you for submitting this PR!

While trying out this script command I ran into some strange errors that were being caused by the file encoding. Could you maybe change the encoding to `utf-8`? It currently shows up as `Mac Roman` for me, and Raycast is unable to parse the script command.

I'm also wondering whether we could make some changes to this script to maybe make adopting it easier. I'm not an AppleScript expert, but I noticed that in other templates we use variables. Have you considered setting variables for `<display>` and `<degree>`, like this:

```applescript
set Display to "LG HDR 4K (1)"
set Rotation to "180°"
```

**Comment by ybbond** [comment]
hi @RSO, thank you for the heads up! Indeed I tried to open with Sublime Text and the encoding detected as UTF-16.
I `touch`ed new UTF-8 file, and copied the whole script to Sublime Text.

I also implemented your suggestion and created a variable for `displayName` and `targetDegree`. Thanks :D

Please try the script again, notice that it's better to edit it in plain text editor than in Script Editor, because of Raycast' issue with emoji 🖥 as icon for AppleScript.

File: commands/system/toggle-flip-screen.template.applescript [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            click
+            click menu item "Standard" of menu 1
+          end if
+        end tell
+      else
+        tell pop up button 1 of tab group 1
+          if (value) contains "Standard" then
+            set currentAction to "Flipping"
+            click
+            click menu item "90°" of menu 1
```
Comment (RSO) [inline commands/system/toggle-flip-screen.template.applescript:68]: Shouldn't this be `targetDegree`?

**Review by RSO — COMMENTED** [review]
Thanks for the fixes, one small comment, the rest looks good!

File: commands/system/toggle-flip-screen.template.applescript (hunk 1 above)
Comment (ybbond) [reply commands/system/toggle-flip-screen.template.applescript]: nice catch. thank you!

**Comment by ybbond** [comment]
Whoops, I missed that line. Nice catch, I pushed new commit to fix that.
Please rereview 🙇

**Review by RSO — APPROVED** [review]

**Pull request merged**